src/command.cpp

```diff
@@ -400,12 +400,15 @@ uint8_t Command::call(const uint8_t device_type, const char * cmd, const char *
         LOG_WARNING(error);
     } else {
         if (single_command) {
-            LOG_DEBUG(("%sCalling command %s"), ro.c_str(), info_s);
+            // TODO not sure if these should go to INFO as there may be a lot of calls, e.g. ioBroker
+            // TODO for now using debug (regarles if compiled with EMSESP_DEBUG)
+            // TODO mvdp ?
+            logger_.debug(("%sCalled command %s"), ro.c_str(), info_s);
         } else {
             if (id > 0) {
-                LOG_INFO(("%sCalling command %s with value %s and id %d on device 0x%02X"), ro.c_str(), info_s, value, id, device_id);
+                logger_.debug(("%sCalled command %s with value %s and id %d on device 0x%02X"), ro.c_str(), info_s, value, id, device_id);
             } else {
-                LOG_INFO(("%sCalling command %s with value %s"), ro.c_str(), info_s, value);
+                logger_.debug(("%sCalled command %s with value %s"), ro.c_str(), info_s, value);
```

Comment:
single command will always reply with data, so debug level is right.
Calling setting commands with value should have info log level. We can see in the log if users flood the bus by sending a lot of commands.